src/components/FilterForm/FormFields.tsx

```diff
@@ -0,0 +1,42 @@
+import { ReactElement } from 'react';
+import { ControllerRenderProps } from 'react-hook-form';
+import { Checkbox } from '@mui/material';
+
+import { FilterFormValuesType } from './types';
+
+// ----- FIELDS ----- //
+
+export const SortingOptionCheckbox = ({
```

Comment:
Do you think it would be possible to consolidate (or refactor) these Checkbox render functions given the checkboxes that are returned is again pretty similar?